fileSystem/views.py

```diff
@@ -181,13 +181,14 @@ def on_completion(self, uploaded_file, request):
 
         f = file.objects.create(name=name, upload=path)
         f.save()
+        serializer = fileSerializer(f)
+        self.response = serializer.data
 
         SaveFile(path, uploaded_file)
         pass
```

Comment:
Remove in next refactor.
The pass statement is a null operation; nothing happens when it executes. The pass is also useful in places where your code will eventually go, but has not been written yet